test/test_pynisher.py

```diff
@@ -517,29 +523,5 @@ def test_capture_output_error(self):
         self.assertIsNone(return_value)
 
 
-@pytest.mark.parametrize("memory_limit", [1024, 2048, 4096])
```

Comment:
Hey, could you please briefly explain why you're moving this test?

Reply:
This PR is based on the [HEAD^](https://github.com/automl/pynisher/tree/0e238554e087dd2778c32834149d751f70194590) and the previous PR by Francisco is completely included in this PR, so I merged the HEAD by overwriting with this PR.
Since this PR is based on the HEAD^, the current test is also checking various memory size, but it is not by parametrization.
I do not mind either way and I can change the corresponding parts.

Reply:
This sounds complicated. Would you mind rebasing your changes on the current head to simplify things?

Reply:
Sorry, I checked the HEAD again and I think it is better to stick to the HEAD^ (i.e. my current PR) version for the following reasons:
1. The current PR changes in test only one line from the previous PR (the version before the latest Francisco's PR)
2. The current PR uses only `unittest` and simpler while Francisco's change adds `pytest` module and requires a new `logger`
3. The previous version and the latest version does not make any differences after all

Note: I finished rebasing.